crates/holochain/src/conductor/conductor/graft_records_onto_source_chain.rs

```diff
@@ -116,6 +116,13 @@ pub(crate) async fn graft_records_onto_source_chain(
             &space.dht_query_cache,
         )
         .await?;
+
+        // Any ops that were moved to the dht_db but had dependencies will need to be integrated.
+        conductor
```

Comment:
This is actually a bit broad, ideally we'd know whether the flush had moved any authored ops to the DHT. It feels wrong to push that logic as deep as the details of the flush... but that would be the most correct. it also feels a bit messy to have to return that information, but that would impact the flush logic least. 

In the worst case, the extra triggers aren't doing too much work in the integrate workflow... but I would like to either deal with this now or come back to it.

Reply:
This seems to be a similar case to the zome init workflow where you're triggering integration afterwards. That part is opaque to me. You're saying that previously it was a chain reaction of sys validation, app validation and then integration. Now the triggers are conditional based on whether work has been done. But why does that lead to manually triggering integration?

Reply:
Yes, your explanation makes more sense. I remember seeing the integration trigger in the call zome workflow. The specs differ from this process, which is why I agree we should revise this.